src/js/demo.js

```diff
@@ -14,8 +14,8 @@ new Vue({
 		rules: {
 			url: 'url',
 			dateISO: 'dateISO',
-			integer: 'integer',
-			numeric: 'numeric',
+			integer: 'integer,min,3,max,10',
```

Comment:
it won't because that rules just applies for numbers and the others do not have the second parameter.